Soccar [UI]/src/nl/soccar/ui/fx/FXMLConstants.java

```diff
@@ -22,6 +22,16 @@
     public static final String WARNING_NEW_USERNAME = "REGISTER USERNAME";
     public static final String MESSAGE_USERNAME_EXISTS = "LOGIN";
     public static final String MESSAGE_NEW_USERNAME = "REGISTER";
+    public static final String MESSAGE_INVALID_USERNAME = "A username can only contain alphanumeric characters & can't contain more than 16 characters.";
+    public static final String MESSAGE_INVALID_USER_PASSWORD = "A password needs to be at least 8 characters long.";
+    public static final String MESSAGE_FALSE_USER_PASSWORD = "The credentials you've provided seem to be incorrect.";

```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Remove this hard-coded password. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS2068)